### PR TITLE
[dash]: Add DPU kvm topology guidance

### PR DESCRIPTION
**Pull request by Pterosaur**

### CODE REVIEW

File: doc/dash/dash-sonic-kvm.md [hunk 1]
```diff
@@ -87,3 +93,90 @@ graph TD
...
+# Run below commands on DUT
+docker exec gnmi supervisorctl stop gnmi-native
+docker exec gnmi bash -c "/usr/bin/nohup /usr/sbin/telemetry -logtostderr --port 50052 --server_crt /etc/sonic/telemetry/gnmiserver.crt --server_key /etc/sonic/telemetry/gnmiserver.key --ca_crt /etc/sonic/telemetry/gnmiCA.pem -gnmi_native_write=true -v=10 >/root/gnmi.log 2>&1 &"
+```
+
+Follow below example to run GNMI client:
+
+```
+# /root/update1 and /root/update2 are protobuf value for DASH table
+python2 /root/gnxi/gnmi_cli_py/py_gnmicli.py --timeout 30 -t 10.0.0.88 -p 50052 -xo sonic-db -rcert /root/gnmiCA.pem -pkey /root/gnmiclient.key -cchain /root/gnmiclient.crt -m set-update --xpath  /APPL_DB/localhost/DASH_APPLIANCE_TABLE[key=123] /APPL_DB/localhost/DASH_VNET_TABLE[key=Vnet1] --value  $/root/update1 $/root/update2
```
Comment (qiluo-msft) [inline doc/dash/dash-sonic-kvm.md:177]: >python2 [](http://example.com/codeflow?start=0&length=7)

Possible to use python3? python2 is deprecated.

File: doc/dash/dash-sonic-kvm.md [hunk 2]
```diff
@@ -87,3 +93,90 @@ graph TD
...
+sshpass -p 'password' ssh -o TCPKeepAlive=yes -o ServerAliveInterval=30 -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no  admin@10.250.0.101
+
+```
+
+## 5.1.3 gnmi client access
+
+Once dash test cases are completed, we can find required certificates in sonic-mgmt container.
+
+```
+# The certificates are in /tmp directory
```
Comment (qiluo-msft) [inline doc/dash/dash-sonic-kvm.md:148]: >tmp [](http://example.com/codeflow?start=27&length=3)

/tmp will be cleaned after reboot. Does it matter?

**Review by r12f — APPROVED** [review]
no more comment besides Qi's. Lgtm.

File: doc/dash/dash-sonic-kvm.md (hunk 1 above)
Comment (Pterosaur) [reply doc/dash/dash-sonic-kvm.md:177]: @ganglyu

File: doc/dash/dash-sonic-kvm.md (hunk 2 above)
Comment (Pterosaur) [reply doc/dash/dash-sonic-kvm.md:148]: @ganglyu

File: doc/dash/dash-sonic-kvm.md (hunk 1 above)
Comment (ganglyu) [reply doc/dash/dash-sonic-kvm.md:177]: py_gnmicli.py is using python2, need to rewrite this tool.

File: doc/dash/dash-sonic-kvm.md (hunk 2 above)
Comment (ganglyu) [reply doc/dash/dash-sonic-kvm.md:148]: Now we generate new certificate for dash test, if there's any requirement, we can use fixed certificate.

Comment (qiluo-msft) [reply doc/dash/dash-sonic-kvm.md:148]: Let's be frank and warn the reader if it has possibility to happen. Another option is to use fixed certificate.

Comment (qiluo-msft) [reply doc/dash/dash-sonic-kvm.md:148]: Feel free to use another PR to resolve this comment.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**